CONTRIBUTING.md

```diff
@@ -4,16 +4,14 @@ Below are the instructions for making changes to simple-chat. There are a few im
 These will make it easier for you to contribute and for us to follow. Thank you for contributing!
 
 ## Getting Started
-If you've found a bug and don't have a solution or have a feature request, please [create an issue](https://github.com/mbrandau/simple-chat/issues/new).
-
-Ensure the bug was not already reported by searching on GitHub under [Issues](https://github.com/mbrandau/simple-chat/issues).
-
-If you're unable to find an open issue addressing the problem, open a new one. Be sure to include a title and clear description, as much relevant information as possible and follow these steps:
```

Comment:
Why remove that completely? I really liked that :smile:

Reply:
oh sorry. Check the latest PR. :)